StitchCoreServicesMongoDbRemote/Sources/StitchCoreServicesMongoDbRemote/RemoteUpdateOptions.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+
+/// Options to use when executing an `updateOne` or `updateMany` command on a `RemoteMongoCollection`.
+public struct RemoteUpdateOptions {
+    /// When true, creates a new document if no document matches the query.
+    public let upsert: Bool?
+    
+    /// Convenience initializer allowing any/all parameters to be optional.
+    public init(upsert: Bool? = nil) {
+        self.upsert = upsert
+    }
+    
+    // Encode everything
+    internal enum CodingKeys: String, CodingKey {
```

Comment:
[minor] These `CodingKeys` aren't needed, and RemoteUpdateOptions should be `Codable`.